app/views/categories/show.html.erb

```diff
@@ -0,0 +1,23 @@
+<div id="expenses-page">
+  <%= render 'layouts/header', title: 'Expenses', back_url: categories_path, logout: true %>
+  <div class="content">
+    <div id="category-container">
+      <div class="category">
+        <img src="<%= @category.icon %>" alt="Category icon">
+        <div class="category-text-container">
+          <h2><%= @category.name %></h2>
+          <p>Total amount spent: <strong>$<%= @category.expenses.sum(:amount) %></strong></p>
+        </div>
+        <%= button_to 'Delete this Category', @category, method: :delete, class: 'btn btn-warning' %>
+      </div>
+    </div>
+
+    <p id="list-of-expenses">List of Expenses</p>
+    <div id="expenses">
+      <% @expenses.each do |expense| %>
+        <%= render expense %>
+      <% end %>
```

Comment:
- [ ] For a given category, the list of transactions(in your case i.e. expenses) should be presented ordered by the most recent.